src/components/common/MainMenu.js

```diff
@@ -23,13 +23,18 @@ const MainMenu = () => {
     isLoading: rootItemIsLoading,
     isError: rootItemIsError,
   } = useItem(rootId);
+
+  const isFolder = Boolean(
+    rootItem && rootItem.get('type') === ITEM_TYPES.FOLDER,
```

Comment:
Do we need to do `?.get` or does that not happen?

Reply:
The first `rootItem &&` will catch if `rootItem` is `null` or `undefined`, so the optional chaining would be redundant.
The operator cannot prevent `rootItem` to not have the `get` method. For instance in the following snippet:

```js
x = {
a: () => 2
};

x && x?.a() && x?.b()
```

will still throw `TypeError: x.b is not a function`